grakn-kb/src/main/java/ai/grakn/kb/internal/ValidateGlobalRules.java

```diff
@@ -276,29 +276,27 @@ private ValidateGlobalRules() {
     }
 
     /**
-     * @param graph graph used to ensure the {@link Relationship} is unique
      * @param relationReified The {@link Relationship} whose hash needs to be set.
      * @return An error message if the {@link Relationship} is not unique.
      */
-    static Optional<String> validateRelationIsUnique(GraknTxAbstract<?> graph, RelationshipReified relationReified){
+    static Optional<String> validateRelationIsUnique(RelationshipReified relationReified){
```

Comment:
Since we're doing the renaming thing - can we make these relation-ship now? 👍

Reply:
Yep. . .also. . . I hate you.